test/bitmap-file-helper-test.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fileHelper = require('../lib/bitmap-file-helper.js');
+
+describe('File Reader Module', function(){
+  describe('with an improper file path',function(){
+    it('should return an error', function(done) {
+      fileHelper('img/badPath.bmp', 'palette', 'invertTransform', function(err){
```

Comment:
@brials @remilonwheels @caylazabel as stated in your `bitmap-file-helper`, excellent pattern of wrapping your file reader so that you can test - great job writing this particular test to check for errors